Modernize documentation (#618)
doctrine/DoctrineMongoDBBundle: Resources/doc/config.rst

.. tip::

    If each environment requires a different MongoDB connection URI, you can

>     `define it as environment variable`_ and reference it in the bundle config:

Reviewer: ```suggestion
    `define it as an environment variable`_ and reference it in the bundle config:
```

  this path is relative it is assumed to be relative to the bundle root. This
  only works if the name of your mapping is a bundle name. If you want to use
  this option to specify absolute paths you should prefix the path with the

>   kernel parameters that exist in the DIC (for example %kernel.project_dir%).

Reviewer: Should we put %...% in backticks so it looks like a code?

It is possible to connect to several mongodb servers on one connection if
you are using a replica set by listing all of the servers within the connection

> string as a comma separated list and using ``replicaSet`` option.

Reviewer: "and" made me think the options is something more
```suggestion
string as a comma separated list provided with the ``replicaSet`` option.
```

            # ...
            connections:
                default:

>                     server: "mongodb://mongodb-01:27017,mongodb-02:27017,mongodb-03:27017/?replicaSet=replSetName"

Reviewer: @alcaeus is this necessary?

Reviewer 2: Yes, if you are connecting to a replica set. See the corresponding [documentation](https://docs.mongodb.com/manual/reference/connection-string/#replica-set-option).
